go.work.sum

```diff
@@ -10,17 +10,32 @@ github.com/gobuffalo/here v0.6.0/go.mod h1:wAG085dHOYqUpf+Ap+WOdrPTp5IYcDAs/x7PL
 github.com/googleapis/gnostic v0.1.0/go.mod h1:sJBsCZ4ayReDTBIg8b9dl28c5xFWyhBTVRp3pOg5EKY=
```

Comment:
Not sure why this is showing up, I tried checking it out to master, and even just copying the contents of master here. Will need a hand debugging.